packages/phoenix-evals/src/phoenix/evals/classify.py

```diff
@@ -268,6 +270,126 @@ def _run_llm_classification_sync(input_data: pd.Series[Any]) -> ParsedLLMRespons
+            Default False.
+
+        use_function_calling_if_available (bool, default=True): If True, use function
+            calling (if available) as a means to constrain the LLM outputs.
+            With function calling, the LLM is instructed to provide its response as a
+            structured JSON object, which is easier to parse.
+
+        provide_explanation (bool, default=False): If True, provides an explanation
+            for each classification label. A column named `explanation` is added to
+            the output dataframe.
+
+        include_prompt (bool, default=False): If True, includes a column named `prompt`
+            in the output dataframe containing the prompt used for each classification.
+
+        include_response (bool, default=False): If True, includes a column named `response`
+            in the output dataframe containing the raw response from the LLM.
+
+        max_retries (int, optional): The maximum number of times to retry on exceptions.
+            Defaults to 10.
+
+        exit_on_error (bool, default=True): If True, stops processing evals after all retries
+            are exhausted on a single eval attempt. If False, all evals are attempted before
+            returning, even if some fail.
+
+        run_sync (bool, default=False): If True, forces synchronous request submission.
+            Otherwise evaluations will be run asynchronously if possible.
+
+        concurrency (Optional[int], default=None): The number of concurrent evals if async
+            submission is possible. If not provided, a recommended default concurrency is
+            set on a per-model basis.
+
+        progress_bar_format(Optional[str]): An optional format for progress bar shown. If not
+            specified, defaults to: llm_classify |{bar}| {n_fmt}/{total_fmt} ({percentage:3.1f}%) "
+            "| ⏳ {elapsed}<{remaining} | {rate_fmt}{postfix}". If 'None' is passed in specifically,
+            the progress_bar log will be disabled.
+
+    Returns:
+        pandas.DataFrame: A dataframe where the `label` column (at column position 0) contains
+            the classification labels. If provide_explanation=True, then an additional column named
+            `explanation` is added to contain the explanation for each label. The dataframe has
+            the same length and index as the input dataframe. The classification label values are
+            from the entries in the rails argument or "NOT_PARSABLE" if the model's output could
+            not be parsed. The output dataframe also includes three additional columns in the
+            output dataframe: `exceptions`, `execution_status`, and `execution_seconds` containing
+            details about execution errors that may have occurred during the classification as well
+            as the total runtime of each classification (in seconds).
+    """
+    if not isinstance(dataframe, pd.DataFrame):
+        dataframe = pd.DataFrame(dataframe, columns=["audio_url"])
```

Comment:
I don't think we should be hard-coding the column name like this maybe? If users are passing in an iterable of urls, we should map the column names to the template variable names